docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -0,0 +1,202 @@
+----
+--
+
+== Take a Single Hot Backup
+--
+. Create the HotBackup resource:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup.yaml[]
+----
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup.yaml
+----
+
+--
+
+== Take Scheduled Hot Backup
+
+You can also take backups on schedule using the `spec.schedule` field.
+--
+. Create the HotBackup resource with `schedule` field filled:
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/hot-backup-scheduled.yaml[]
+----
++
+The field needs to take a valid cron expression. Some examples:
++
+[cols="1,1"]
+|===
+| ```30 10 * * *```
+| On 10:30 AM every day
+
+| ```0, 0, 1,15,25 * *```
+| On 1st, 15th and 25th each month, midnight
+
+| ```@monthly```
+| Once a month, first of the month, midnight
+|===
+
+
+. Apply the resource:
++
+[source,bash]
+----
+kubectl apply -f ./hot-backup-scheduled.yaml
+----
+--
+
+== Restore from HotBackup
+
+. Delete the Hazelcast cluster:
++
+[source,bash]
+----
+kubectl delete hazelcast hazelcast
+----
+
+. Check PVCs are still there:
++
+[source,bash]
+----
+kubectl get pvc -l app.kubernetes.io/instance=hazelcast
+----
++
+[source,bash]
+----
+NAME                                  STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+hot-restart-persistence-hazelcast-0   Bound    pvc-116b4084-a436-4462-b413-511b77df307b   20Gi       RWO            standard       6m
+hot-restart-persistence-hazelcast-1   Bound    pvc-a7711b6b-dcbf-45cb-9577-8ce6b1892f2f   20Gi       RWO            standard       5m
+hot-restart-persistence-hazelcast-2   Bound    pvc-64314d82-da7a-4e38-bd2d-e770a63dc4e8   20Gi       RWO            standard       5m
+
+----
+
+. Reapply the Hazelcast resource:
++
+[source,bash]
+----
+kubectl apply -f ./hazelcast-persistence.yaml
+----
+
+. Same PVCs will be bound to the new pods:
++
+[source,bash]
+----
+kubectl get pods -l app.kubernetes.io/instance=hazelcast -ojsonpath='{.items[*].spec.volumes[?(@.name=="hot-restart-persistence")].persistentVolumeClaim.claimName}'
+
+----
++
+[source,bash]
+----
+hot-restart-persistence-hazelcast-0 hot-restart-persistence-hazelcast-1 hot-restart-persistence-hazelcast-2
+----
+
+=== Restore Policy
+
+Restore policy can be adjusted by setting `clusterDataRecoveryPolicy` field. Supported
+options are:
+
+[cols="1,1"]
+|===
+| ```FullRecoveryOnly```
+| Does not allow partial start of the cluster.
+
+| ```PartialRecoveryMostRecent```
+| Allows partial start with the members that have most recent partition table.
+
+| ```PartialRecoveryMostComplete```
+| Allows partial start with the member that have most complete partition table.
+
+| ```ForceStart```
+| Deletes all data in the cluster persistence store if cluster recovery fails.
+More detail can be found in link:https://docs.hazelcast.com/hazelcast/5.1/storage/triggering-force-start[Hazelcast documentation].
+|===
+
+== HostPath Support for Persistence
```

Comment:
This is an option for enabling Persistence. I think it should go under 'Enabling Persistence' and we should add a description to discuss the differences (pros/cons) between using HostPath and PVC.

Reply:
The reason that I moved it to the end of the page is actually to hide it. We support it but we do not incentivize it. 
@hasancelik What do you think?

Reply:
What's the use case for it? If we have it, we must have users who want it, and those users will expect a description of it in the right place (enabling persistence).

Reply:
@hakanmemisoglu @hasancelik, maybe it's better to add a note on why this option is not recommended? Since we support this option, I think it's better to just warn the users of consequences but not hide it

Reply:
@JakeSCahill, @hakanmemisoglu is right, `hostPath` option is not a complete replacement for `PVC`. We support this option for some specific use cases so promoting it like replacement of PVC based which is recommended way does not make sense.

Reply:
Actually, it would be repeating what is mentioned in Kubernetes documentation (which I gave a link to).